### ISSUE TITLE
Latest 8.2.0 FOSS release broken

### ISSUE THREAD

**Issue by cryzed** [tigase/tigase-server#142]
**Describe the bug**
The latest 8.2.0 FOSS release is broken. When running the unmodified `-dist` release using `scripts/tigase.sh run etc/tigase.conf` and following the setup, the following exception occurs when attempting to complete the "Database configuration"-step:

```
[2022-03-20 18:12:14:945] [WARNING ] [    http-server-pool-4 ] DependencyManager.createBeanConfig(): Cannot create bean config 'push', type=tigase.push.PushNotificationsComponent. Bean requires unknown class tigase/licence/LicenceCheckerUpdateCallback
[2022-03-20 18:12:14:945] [SEVERE  ] [    http-server-pool-4 ] SetupServlet.service()           : exception processing request
java.lang.NullPointerException: Cannot invoke "tigase.kernel.core.BeanConfig.setExportable(boolean)" because "this.beanConfig" is null
        at tigase.kernel.core.BeanConfigBuilder.exportable(BeanConfigBuilder.java:124)
        at tigase.kernel.core.Kernel.registerBean(Kernel.java:501)
        at tigase.kernel.beans.config.AbstractBeanConfigurator.lambda$registerBeans$3(AbstractBeanConfigurator.java:534)
        at java.base/java.util.HashMap.forEach(HashMap.java:1421)
        at tigase.kernel.beans.config.AbstractBeanConfigurator.registerBeans(AbstractBeanConfigurator.java:514)
        at tigase.db.util.SchemaManager.getRepositoryBeans(SchemaManager.java:958)
        at tigase.db.util.SchemaManager.getDataSourcesAndSchemas(SchemaManager.java:436)
        at tigase.db.util.SchemaManager.loadSchemas(SchemaManager.java:465)
        at tigase.http.modules.setup.pages.DBCheckPage.loadSchema(DBCheckPage.java:51)
        at tigase.http.modules.setup.pages.DBCheckPage$loadSchema.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
        at groovy.tmp.templates.GStringTemplateScript370$_getTemplate_closure1.doCall(GStringTemplateScript370.groovy:25)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
        at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
        at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
        at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
        at groovy.lang.Closure.call(Closure.java:412)
        at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:912)
        at tigase.http.modules.setup.SetupServlet.processRequest(SetupServlet.groovy:138)
        at tigase.http.modules.setup.SetupServlet.service(SetupServlet.groovy:69)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at tigase.http.java.filters.DummyFilterChain.doFilter(DummyFilterChain.java:48)
        at tigase.http.java.filters.ProtocolRedirectFilter.doFilter(ProtocolRedirectFilter.java:69)
        at tigase.http.java.filters.DummyFilterChain.doFilter(DummyFilterChain.java:46)
        at tigase.http.java.RequestHandler.handle(RequestHandler.java:167)
        at jdk.httpserver/com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:95)
        at jdk.httpserver/sun.net.httpserver.AuthFilter.doFilter(AuthFilter.java:82)
        at jdk.httpserver/com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:98)
        at jdk.httpserver/sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(ServerImpl.java:733)
        at jdk.httpserver/com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:95)
        at jdk.httpserver/sun.net.httpserver.ServerImpl$Exchange.run(ServerImpl.java:700)
        at tigase.http.java.JavaStandaloneHttpServer$ExecutorWithTimeout.lambda$0(JavaStandaloneHttpServer.java:195)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

**To Reproduce**
1. Download [tigase-server-8.2.0-b11690-dist.tar.gz](https://github.com/tigase/tigase-server/releases/download/tigase-server-8.2.0/tigase-server-8.2.0-b11690-dist.tar.gz)
2. Extract the archive
3. Run `scripts/tigase.sh run etc/tigase.conf` from inside the folder
4. Follow the setup with default values at `http://localhost:8080` until the "Database configuration"-step
5. Attempt to continue to the next step

**Impact**
Tigase is unusable.

**Expected behavior**
Tigase is usable.

**Details (please complete the following information):**
 - Tigase version: 8.2.0
 - JVM flavour and version: openjdk 17.0.3 2022-04-19
 - Operating system/distribution/version: Arch Linux

**Additional context**
The Docker releases on Dockerhub are also broken due to the same reason. The enterprise edition seems to work fine.

**Comment by woj-tek** [comment]
Thank you for reporting it, we will fix it as soon as possible.

(ref #serverdist-11)

**Comment by woj-tek** [comment]
issue was caused by inclusion of tigase-push component in dist package. The fix is planned for 8.3 release.

**Comment by woj-tek** [comment]
Version 8.3 has been released and issue is fixed now.
